### PR TITLE
Fix invoke override

### PR DESCRIPTION
**Pull request by MilGard91**
- Adding options extraction in build invoke function

### CODE REVIEW

File: src/contract/default.ts [hunk 1]
```diff
@@ -46,7 +46,18 @@ function buildCall(contract: Contract, functionAbi: FunctionAbi): AsyncContractF
...
+      if (!/_len$/.test(input.name)) {
+        return acc + 1;
+      }
+      return acc;
+    }, 0);
+    const options = {};
+    if (inputsLength + 1 === args.length && typeof args[args.length - 1] === 'object') {
+      Object.assign(options, args.pop());
+    }
+    return contract.invoke(functionAbi.name, args, options);
```
Comment (janek26) [inline src/contract/default.ts:60]: why was this added?

File: src/utils/stark.ts [hunk 2]
```diff
@@ -48,3 +49,8 @@ export function compileCalldata(args: RawArgs): Calldata {
     return toBN(value).toString();
   });
 }
+
+export function estimatedFeeToMaxFee(estimatedFee: BigNumberish, overhead: number = 0.15): BN {
+  const fee = isBN(estimatedFee) ? estimatedFee : toBN(estimatedFee);
```
Comment (janek26) [inline src/utils/stark.ts:54]: no check needed, if you pass a BN to `toBN` it'll just return that BN

File: src/contract/default.ts (hunk 1 above)
Comment (MilGard91) [reply src/contract/default.ts:60]: it checks if the last arg when invoking is the override object.

Example: contract.methodName(...args, {maxFee: '0'})
